app_dart/README.md

```diff
@@ -80,13 +85,30 @@ To update the cron tasks in the App Engine project, run:
 $ gcloud app deploy cron.yaml
 ```
 
-### ssh into instance
+### Local development
+
+#### Using physical machine
+
+* Starting server
+
+```sh
+export COCOON_USE_IN_MEMORY_CACHE=true
+dart bin/server.dart
+```
+
+If you see Serving requests at 0.0.0.0:8080 the dev server is working.
+You should also set `COCOON_USE_IN_MEMORY_CACHE=true` as you typically
+don't have access to the remote redis instance during local development.
+
+#### Using Docker
+
+* ssh into instance
```

Comment:
Should this be "Starting docker instance"?

Reply:
This is not starting the instance.